sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuDeviceManager.scala

```diff
@@ -305,6 +305,63 @@ object GpuDeviceManager extends Logging {
     }
   }
 
+  /**
+   * Parse the rmm allocation mode from the configs.
+   *
+   * There are two relevant configs, "isPooledMemEnabled" and "rmmPool".
+   * The former one is deprecated, and the later "rmmPool" is recommended. So
+   * "rmmPool" will always be respected when they conflict with each other.
```

Comment:
Oops, missed this before. We should update this doc as well.

Reply:
done